extensions/amp-viz-vega/0.1/amp-viz-vega.js

```diff
@@ -137,20 +175,54 @@ export class AmpVizVega extends AMP.BaseElement {
    */
   renderGraph_() {
     return new Promise((resolve, reject) => {
-      vega.parse.spec(this.data_, (error, chart) => {
+      this.vega_.parse.spec(this.data_, (error, chartFactory) => {
```

Comment:
`onLayoutMeasure` doesn't calls `renderGraph` until `this.graph_` is set which is guaranteed to be after the first layout (alternatively I could have checked `this.firstLayoutCompleted_` in `onLayoutMeasure` but it is private)